cache/manager.go

```diff
@@ -1458,64 +1471,77 @@ const (
 	cachePolicyRetain
 )
 
-type noUpdateLastUsed struct{}
+type NoUpdateLastUsedOpt struct{}
 
-var NoUpdateLastUsed noUpdateLastUsed
+var (
+	NoUpdateLastUsed  RefOption = NoUpdateLastUsedOpt{}
+	CachePolicyRetain RefOption = CachePolicyRetainOpt{}
+)
 
-func CachePolicyRetain(m *cacheMetadata) error {
-	return m.SetCachePolicyRetain()
-}
+type CachePolicyRetainOpt struct{}
 
-func CachePolicyDefault(m *cacheMetadata) error {
-	return m.SetCachePolicyDefault()
+func CachePolicyDefault() RefOption {
```

Comment:
I noticed `CachePolicyDefault` was unused.  I could totally remove it if you would like